src/main/java/persistence/sql/dml/DmlQueryBuilder.java

```diff
@@ -1,37 +1,65 @@
 package persistence.sql.dml;
 
+import persistence.sql.metadata.ColumnData;
+import persistence.sql.metadata.ColumnDatas;
+import persistence.sql.metadata.EntityData;
 import persistence.sql.metadata.EntityMetadata;
 
-public class DmlQueryBuilder<T> {
+import java.util.List;
 
-    private final EntityMetadata<T> entityMetadata;
+public class DmlQueryBuilder {
 
-    private DmlQueryBuilder(EntityMetadata<T> entityMetadata) {
-        this.entityMetadata = entityMetadata;
+    public String buildInsertQuery(Object entity) {
+        EntityData entityData = EntityData.from(entity);
+        ColumnDatas insertColumns = entityData.getInsertColumns();
+        return "insert into " + entityData.getTableName() + " (" + String.join(", ", insertColumns.getColumnNames()) + ")"
+                + " values (" + String.join(", ", insertColumns.getColumnValues()) + ")"
+                + ";";
     }
 
-    public static <T> DmlQueryBuilder<T> from(Class<T> entityClass) {
-        return new DmlQueryBuilder<>(EntityMetadata.from(entityClass));
+    public String buildSelectAllQuery(Class<?> entityClass) {
+        EntityMetadata metadata = EntityMetadata.from(entityClass);
+
+        return "select *"
+                + " from " + metadata.getTableName()
+                + ";";
     }
 
-    public String buildInsertQuery(T entity) {
-        return "insert into " + entityMetadata.getTableName() + " (" +
-                String.join(", ", entityMetadata.getInsertColumnNames()) +
-                ") values (" +
-                String.join(", ", entityMetadata.getInsertColumnValues(entity)) +
-                ");";
+    public String buildSelectByIdQuery(Class<?> entityClass, Object id) {
+        EntityMetadata metadata = EntityMetadata.from(entityClass);
+
+        return "select *"
+                + " from " + metadata.getTableName()
+                + " where " + metadata.getPrimaryKeyName() + " = " + id
+                + ";";
```

Comment:
Java15 에 추가된 [Text block](https://www.baeldung.com/java-text-blocks) 을 이용해봐도 좋을 것 같아요 :)
```suggestion
        return """
                select * \
                from %s \
                where %s = %s;\
                """.formatted(metadata.getTableName(), metadata.getPrimaryKeyName(), id);
```

Reply:
클래스로 만들자니 과한 것 같고, 텍스트 블록으로 만들자니 %s가 명확하지 않고 치환을 누락해도 컴파일 타임에 오류 발견이 어려워 계속 고민하다 이상태가 됐습니다 ㅎㅎ  

텍스트 블록으로 변경했습니다~